src/test/scala/org/apache/spark/sql/BebeFunctionsSpec.scala

```diff
@@ -110,6 +110,48 @@ class BebeFunctionsSpec
     }
   }
 
+  describe("endOfDay") {
```

Comment:
Can you add an example when this function is run on a DateType column?  It should return a TimestampType column.